### ISSUE TITLE
Yatopia 1.16.2 latest version, Portal bug

### ISSUE THREAD

**Issue by P-333** [tr7zw/YAPFA#174]
**Expected behavior**
When you go in to a portal it doesn't generate a new one if a portal exists on the other side.

**Observed behavior**
When you go through a portal and in 1.16.1 it already generated one. It still generates a new portal.

**Steps to reproduce (if known)**
You generate a portal to the nether in 1.16.1. Then in 1.16.2 when you try to use the same one. It still generates a new one.

**Version**
![image](https://user-images.githubusercontent.com/49524570/91635208-d69c1000-e9f6-11ea-9d15-f6acd958139d.png)

**Additional context**
Nope not really. I can add tho for joke purposes only. That potatoes is life.

**Comment by P-333** [comment]
This only happens when using Yatopia and not with Tuinity or paper.

**Comment by toinouH** [comment]
Note: the world was not forceupgraded

Halvnaken#5585 had the same issue

**Comment by MrIvanPlays** [comment]
use portal distance option is true or false

**Comment by P-333** [comment]
use-block-distance-in-portal-search-radius: false

**Comment by MrIvanPlays** [comment]
another issue with village place optimizations which bgidiere merged accidentally yesterday. use build which does not have them is the workaround rn.

**Comment by TheFaser** [comment]
It's true. I also had such a problem with portals on an earlier version of yatopia, on the latter it has not yet been observed

**Comment by toinouH** [comment]
https://github.com/YatopiaMC/Yatopia/actions/runs/229054221
Can you try with that ? It's just before the pull request was merged

**Comment by TheFaser** [comment]
This bug occurred when the kernel was changed and only once. Now on the last yatopia I did not notice this

**Comment by MrIvanPlays** [comment]
fixed on latest
